driver-embedded/src/main/com/mongodb/embedded/client/MongoDBCAPIHelper.java

```diff
@@ -165,4 +171,48 @@ private static void checkInitialized() {
 
     private MongoDBCAPIHelper() {
     }
+
+
+    public static class MongoDBCAPIInitParams extends Structure implements MongoDBCAPI.InitParams {
```

Comment:
`MongoDBCAPIInitParams` represents the struct in c: 
```typedef struct {
    const char* yaml_config;
    uint64_t log_flags;
    libmongodbcapi_log_callback log_callback;
    void* log_user_data;
} libmongodbcapi_init_params;
```

The fields are required to be public by JNA - hence the checkstyle and spotbugs exclusions.  The fields are reflected upon via the required `getFieldOrder` method.